### ISSUE TITLE
[enhance] pouch info takes too long due to local-persist plugin

### ISSUE THREAD

**Issue by Letty5411** [alibaba/pouch#1396]
### Ⅰ. Issue Description
pouch info may cost about 10+ seconds due to activating local-persist plugin failed.

### Ⅱ. Describe what happened
```
time="2018-05-24T11:10:57.982808324+08:00" level=debug msg="Calling GET /v1.24/info, client @"
time="2018-05-24T11:10:57.98290344+08:00" level=debug msg="run command: [uname] [-r]"
time="2018-05-24T11:10:57.983595981+08:00" level=debug msg="Get a grpc client" elapsed=2.815µs
time="2018-05-24T11:10:57.984313732+08:00" level=warning msg="plugin unix:///run/docker/plugins/local-persist.sock call /Plugin.Activate failed, retry after 1000000000 seconds"
time="2018-05-24T11:10:58.984483179+08:00" level=warning msg="plugin unix:///run/docker/plugins/local-persist.sock call /Plugin.Activate failed, retry after 2000000000 seconds"
time="2018-05-24T11:11:00.984648527+08:00" level=warning msg="plugin unix:///run/docker/plugins/local-persist.sock call /Plugin.Activate failed, retry after 4000000000 seconds"
time="2018-05-24T11:11:04.984950858+08:00" level=warning msg="plugin unix:///run/docker/plugins/local-persist.sock call /Plugin.Activate failed, retry after 8000000000 seconds"
time="2018-05-24T11:11:12.985356324+08:00" level=info msg="End of Calling GET /v1.24/info, costs 15002 ms. client @"
```

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1.
2.
3.

### Ⅵ. Environment:

- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by shaloulcy** [comment]
It's caused by local-persist. When do ci tests, we will start local-persist plugin which will create a unix socket in `/var/run/docker/plugins/local-persist.sock`. But after ci tests end, the local-persisit plugin exited, but the socket still exists. 

`pouch info` will try to communicate with the sock, it will timeout.

So we will do some cleaning

**Comment by allencloud** [comment]
I think this issue has been fixed in pr #1400 , Closing it.
